packages/local_auth/local_auth_darwin/darwin/Tests/FLALocalAuthPluginTests.swift

```diff
@@ -0,0 +1,403 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import local_auth_darwin
+
+// Set a long timeout to avoid flake due to slow CI.
+let timeout: TimeInterval = 30.0
+
+/// A context factory that returns preset contexts.
+class StubAuthContextFactory: NSObject, FLADAuthContextFactory {
```

Comment:
nit: can be `final class` (unless we intend to subclass it).

Reply:
Done.